modules/drivers/athena/test/metabase/test/data/athena.clj

```diff
@@ -58,11 +59,46 @@
   ([_ db-name table-name]            [db-name table-name])
   ([_ db-name table-name field-name] [db-name table-name field-name]))
 
+;;; INSTRUCTIONS FOR MANUALLY DROPPING AND RECREATING A DATABASE
+;;;
+;;; 1. Install the AWS CLI if you haven't done so already
+;;;
+;;; 2. Create a profile using the `MB_ATHENA_TEST_ACCESS_KEY`, `MB_ATHENA_TEST_SECRET_KEY`, and `MB_ATHENA_TEST_REGION`
+;;;    you're using to run tests
+;;;
+;;;    ````
+;;;    aws configure --profile athena-ci
+;;;
+;;; 3. Delete the data from the `MB_ATHENA_TEST_S3_STAGING_DIR` S3 bucket. The data directory is the same as the dataset
+;;;    name you want to delete with hyphens replaced with underscores e.g. `sample-dataset` becomes `sample_dataset`
+;;;
+;;;    ```
+;;;    aws s3 --profile athena-ci rm s3://metabase-ci-athena-results/sample_dataset --recursive
+;;;    ```
+;;;
+;;; 4. Delete the database from the Glue Console.
+;;;
+;;;    ```
+;;;    aws glue --profile athena-ci delete-database --name sample_dataset
+;;;   ```
+;;;
+;;; 5. After this you can recreate the database normally using the test loading code. Note that you must
+;;;    enable [[*allow-database-creation*]] for this to work:
+;;;
+;;;    ```
+;;;    (db/delete! 'Database :engine "athena", :name "sample-dataset")
+;;;    (binding [metabase.test.data.athena/*allow-database-creation* true]
+;;;      (metabase.driver/with-driver :athena
+;;;        (metabase.test/dataset sample-dataset
+;;;          (metabase.test/db))))
+;;;    ```
```

Comment:
I wrote down the steps for manually deleting data from the S3 bucket and Glue Console in case the CI issue ever comes up again. Hopefully not -- this PR should hopefully prevent that.